package.json

```diff
@@ -99,7 +99,7 @@
   },
   "scripts": {
     "test-ci": "gulp ci-tests",
-    "test": "rimraf ./build && tsc && mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 30000  ./build/compiled/test",
+    "test": "rimraf ./build && tsc && mocha --file ./build/compiled/test/utils/test-setup.js  --recursive --timeout 60000 ./build/compiled/test",
```

Comment:
whats wrong with `--bail`?

Reply:
`--bail` makes the tests stop after the first failure. This can be a good thing as tests that run later in the same file seem to all fail anyway due to transaction issues but, on the other hand, it can make finding and fixing test failure across the entire test suite a massive slog as you have to keep running all of the tests over and over each time you fix the latest failure.

Reply:
Ooops, this change happened while I was setting typeorm development environment for checking total failed tests.
it's my commit mistake. Thanks for reviewing!